pkg/api/controller_test.go

```diff
@@ -3404,45 +3404,50 @@ func TestController_ClientDisconnect(t *testing.T) {
 	clt := setupClientByEndpoint(t, server.URL, "", "")
 	_ = setupCommPrefs(t, clt)
 	cred := createDefaultAdminUser(t, clt)
-	clt = setupClientByEndpoint(t, server.URL, cred.AccessKeyID, cred.SecretAccessKey)
 
+	clt = setupClientByEndpoint(t, server.URL, cred.AccessKeyID, cred.SecretAccessKey)
```

Comment:
Not needed

Reply:
The first `clt` (without creds) does setup lakefs.
The second `clt` (this one - with creds) create a repo.

Reply:
Correct me if I'm wrong - but I don't see where clt is used between this line and line 3428